.github/workflows/primer_run_pr.yaml

```diff
@@ -11,6 +11,11 @@ on:
       - "tests/primer/**"
       - "requirements*"
       - ".github/workflows/**"
+      # We ignore these specific files because they need to be changed
+      # on 'main' and will always fail the PR run.
+      - "!.github/workflows/primer_run_main.yaml"
+      - "!.github/workflows/primer_comment.yaml"
```

Comment:
Why will these two always fail? #6724 didn't fail.

Reply:
It passed because there wasn't any difference between using bleeding edge or latest release. `run_main` didn't use bleeding edge yet because the workflow wasn't on `main` yet.

I'd say we don't want to give a false sense of security and just exclude any PRs that only change `main` files.

Reply:
OK to merge -- not worth trying to anticipate more small targeted PRs like #6724 which should be rare.